src/Apps/Order2/Routes/Checkout/Components/DeliveryOptionsStep/Order2DeliveryOptionsStep.tsx

```diff
@@ -0,0 +1,37 @@
+import { Box, Flex, Text } from "@artsy/palette"
+import {
+  CheckoutStepName,
+  CheckoutStepState,
+} from "Apps/Order2/Routes/Checkout/CheckoutContext/types"
+import { useCheckoutContext } from "Apps/Order2/Routes/Checkout/Hooks/useCheckoutContext"
+
+interface Order2DeliveryOptionsStepProps {}
+
+export const Order2DeliveryOptionsStep: React.FC<
+  Order2DeliveryOptionsStepProps
+> = () => {
+  const { steps } = useCheckoutContext()!
```

Comment:
`!` should not be necessary I hope.

Reply:
I keep copying the boilerplate form the payment step and forget to remove it. 😅